packages/workbox-background-sync/test/browser/request-manager.js

```diff
@@ -58,11 +64,31 @@ describe('request-manager test', () => {
   it('check replay', async function() {
     const backgroundSyncQueue
       = new workbox.backgroundSync.test.BackgroundSyncQueue({
-        callbacks,
-      });
+      callbacks,
+    });
     await backgroundSyncQueue.pushIntoQueue({request: new Request('/__echo/counter')});
     await backgroundSyncQueue.pushIntoQueue({request: new Request('/__echo/counter')});
     await backgroundSyncQueue._requestManager.replayRequests();
     chai.assert.equal(responseAchieved, 2);
   });
+
+  it(`will fetch() the request returned by the RequestWrapper's requestWillFetch plugin`, async function() {
+    const stub = sinon.stub(self, 'fetch').returns(new Response());
```

Comment:
Sorry, I missed the linting error. I've added `sinon` to the eslint globals config.